fitbit-react/src/Util/Toasts/toasts.css

```diff
@@ -0,0 +1,18 @@
+.toasts-container{
+    z-index: 1000;
```

Comment:
just a suggestion keep the z-index low well if you need it you can keep it as it is,

Reply:
Yes i need the toasts to appear on top of all